incubator/kafka/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Apache Kafka is publish-subscribe messaging rethought as a distributed
   commit log.
 name: kafka
-version: 0.16.2
+version: 0.16.3
```

Comment:
Should we bump the version to `0.17.0`, because of the PR include the new feature and in a backward-compatible manner?